### PR TITLE
I filled the tables for ingredients, recipes and alternatives / linked My Recipe Button with Modal / created API to fetch all recipes on My Recipe Book page

### PR DESCRIPTION
**Pull request by jcccookie**

### CODE REVIEW

File: database/scripts/ingredientAlternativeData.js [hunk 1]
```diff
@@ -1,4 +1,27 @@
 module.exports.ingredientAlternatives = [
     ['67', '94'],
-    ['67', '95']
+    ['67', '95'],
```
Comment (Jack-Edwards) [inline database/scripts/ingredientAlternativeData.js:3]: I haven't looked at the ingredient alternative code in a while.  Is this list saying ingredient with id=67 has multiple alternatives?

**Comment by Jack-Edwards** [comment]
Looks good to me.  Just the one question that I'm curious about

**Review by Jack-Edwards — APPROVED** [review]

File: database/scripts/ingredientAlternativeData.js (hunk 1 above)
Comment (peterst5) [reply database/scripts/ingredientAlternativeData.js:3]: Yeah, so the ingredients should have MULTIPLE alternatives if possible. The first 2 were ones that I out in to check my modal.

Comment (jcccookie) [reply database/scripts/ingredientAlternativeData.js:3]: Yes, it looks like mozzarella has two alternatives, pasture-raised and vegan mozzarella. I think ingredients can have multiple alternatives.

**Review by peterst5 — CHANGES_REQUESTED** [review]
Each ingredient should have more than 1 alternative if possible.

**Review by peterst5 — APPROVED** [review]

**Pull request merged**